[#40498] Update OpenID documentation (add Google)

adam-op opened the pull request:
[#40498] Update OpenID documentation (add Google)

https://community.openproject.org/work_packages/40498

Commit:
Update README.md

oliverguenther commented on docs/system-admin-guide/authentication/openid-providers/README.md:
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
+1. Navigate to your GCP console.  (https://console.cloud.google.com/)
+2. Go to **APIs & Services** > OAuth consent screen.
+
+![image-20220215095201951](/home/ant0n/.config/Typora/typora-user-images/image-20220215095201951.png)
```
The paths to the images are not correct, they need to be relative to the README.md file, for example `./images/image-20220215095201951` if you put them in a subfolder `images`

You can configure Typora to move the images relative to the file

![image](https://user-images.githubusercontent.com/459462/154044667-af5353a8-11b8-471e-8672-5d31c8fc0698.png)

oliverguenther commented on docs/system-admin-guide/authentication/openid-providers/README.md:
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
+   5. **Developer Contact information** (e.g.  developer@example.com)
+   6. Click **SAVE AND CONTINUE** to proceed.
+
+![image-20220215095953811](/home/ant0n/.config/Typora/typora-user-images/image-20220215095953811.png)
```
Same here

oliverguenther commented on docs/system-admin-guide/authentication/openid-providers/README.md:
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
+
+1. Under **APIs & Services**, go to **Credentials**.
+
+![image-20220215102049207](/home/ant0n/.config/Typora/typora-user-images/image-20220215102049207.png)
```
Same here

oliverguenther commented on docs/system-admin-guide/authentication/openid-providers/README.md:
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
+
+   4. Click **CREATE** or **SAVE** .
+
+![image-20220215102234492](/home/ant0n/.config/Typora/typora-user-images/image-20220215102234492.png)
```
Same here

oliverguenther commented on docs/system-admin-guide/authentication/openid-providers/README.md:
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
+- Note **Client ID**
+- Note **Client Secret**
+
+![image-20220215102658395](/home/ant0n/.config/Typora/typora-user-images/image-20220215102658395.png)
```
Same here

oliverguenther commented on docs/system-admin-guide/authentication/openid-providers/README.md:
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
+   4. **Secret** (**Client Secret** from step 2)
+3. Press **Create**
+
+![image-20220215103036615](/home/ant0n/.config/Typora/typora-user-images/image-20220215103036615.png)
```
Same here

oliverguenther commented on docs/system-admin-guide/authentication/openid-providers/README.md:
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
+
+4. The following green notification **Successful creation** should appear
+
+![image-20220215103344744](/home/ant0n/.config/Typora/typora-user-images/image-20220215103344744.png)
```
Same here

oliverguenther commented on docs/system-admin-guide/authentication/openid-providers/README.md:
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
+
+
+
+
```
Please avoid repeated newlines like that (keep them <= 2 newlines for spacing purposes)

Commit:
changed location for screenshots

1) the locations for screenshots is now in the same location as the README.MD

2) changed blank lines to maximum 2

ghost commented:
@oliverguenther I corrected the requested changes, comitted, pushed origin, what next?
